ChrisAPI_data.js

```diff
@@ -0,0 +1,88 @@
+
+
+// get todo response object
+async function getTodos() {
+    let fromUrl = "https://jsonplaceholder.typicode.com/todos";// 
+    try {
+        let res = await fetch(fromUrl);
+        return await res.json();
+    } catch (error) {
+        console.log(error); // log
+    }
+}
+
+ function renderTodoById(){
+    //let todoCount = 200;   
+    let defaultvalue = parseInt(document.getElementById("hidenNextValue").value); // got value when page loaded
+    let userRequest = document.getElementById("inputId").value;
+    let statusColor = "black"
+    if (userRequest !="")
+    {
+        if( userRequest > 200 ) return; // max exceeded
+    let url = "https://jsonplaceholder.typicode.com/todos/"+ userRequest;
+    fetch(url)
+    .then ( data => data.json())
+    .then( 
+        json => 
+            {
+                document.getElementById('title').value = json.title;
+                document.getElementById("status").value = json.completed;
+
+                json.completed == true? statusColor="green":statusColor ="red";                                       
+                document.getElementById('title').style.color=statusColor;
+                document.getElementById('title').style.fontWeight="bold";
+                document.getElementById("status").style.color=statusColor;
+                document.getElementById('status').style.fontWeight="bold";
+            }
+         )
+
+       } 
+    else // use next button
+        {  
+           
+        defaultvalue = parseInt(document.getElementById("hidenNextValue").value);
+         let url = "https://jsonplaceholder.typicode.com/todos/"+ defaultvalue;
```

Comment:
This is where you can just use a default URL, no need to grab the "1" from the html page.